apps/backend/src/modules/categories/category.repository.ts

```diff
@@ -73,6 +73,17 @@ class CategoryRepository implements Repository {
 			.where({ userId })
 			.delete();
 	}
+	public async deleteUserScoresByCategoryIds(
+		userId: number,
+		categoryIds: number[],
+	): Promise<number> {
+		return await this.categoryModel
+			.query()
+			.from(DatabaseTableName.QUIZ_SCORES)
+			.whereIn("category_id", categoryIds)
```

Comment:
```suggestion
			.whereIn("categoryId", categoryIds)
```
I think this should work